src/ui/layout/qgslayoutlegendwidgetbase.ui

```diff
@@ -1226,9 +1243,9 @@
    <header>qgsfontbutton.h</header>
   </customwidget>
   <customwidget>
-   <class>QgsPropertyOverrideButton</class>
-   <extends>QToolButton</extends>
-   <header>qgspropertyoverridebutton.h</header>
+   <class>QgsLayoutItemComboBox</class>
+   <extends>QComboBox</extends>
+   <header>qgslayoutitemcombobox.h</header>
```

Comment:
Was this change intended?

Reply:
Just the UI builder being weird on windows.